doc/tasks/infection.md

```diff
@@ -0,0 +1,79 @@
+# Infection
+
```

Comment:
Should we suggest to load `infection/infection` trough composer locally / globally ?
In other task documentation, this is added to the top + in the composer.json file under suggestions.

Since the documentation of infection is suggesting global installation, we might only add it in the task documentation?

Reply:
I would suggest to load it locally. I already updated the documentation.

Reply:
Can you add the composer require command to the documentation as well? e.g. https://github.com/phpro/grumphp/blob/master/doc/tasks/phpunit.md